NetCDF/template/global_attributes_timeSeries.txt

```diff
@@ -41,8 +41,8 @@ N, geospatial_lat_max       = [ddb Site Sites Latitude]
 N, geospatial_lon_min       = [ddb Site Sites Longitude]
 N, geospatial_lon_max       = [ddb Site Sites Longitude]
 N, instrument_nominal_height= 
-N, instrument_nominal_depth = 
-N, site_nominal_depth       = 
+N, instrument_nominal_depth = [ddb InstrumentDepth]
+N, site_nominal_depth       = [ddb Site Sites Depth]
 N, site_depth_at_deployment = 
```

Comment:
I'm not comfortable to do that since these are not valid config for everyone at the moment.